parsers/BR.py

```diff
@@ -142,10 +150,16 @@ def fetch_exchange(country_code1, country_code2, session=None):
 
     gd = get_data()
 
+    if country_code1 in countries_exchange.keys():
+        country_exchange = countries_exchange[country_code1]
+
+    if country_code2 in countries_exchange.keys():
+        country_exchange = countries_exchange[country_code2]
+
     data = {
```

Comment:
Are you sure the direction is good? The parser assumes the country2 is importing from country1.

Reply:
I will check it out better

Reply:
The directions are the same for Uruguay and this code don't change the directions at all, just try to find in each parameter if has the key that correspond to the country of exchange with Brasil. Just change this to more generic, because now we have another keys that corresponds to other countries of exchanges with Brazil.

Reply:
I am not sure I completely follow. Are you saying you'll improve the code to set direction based on which countries are passed?
This function is now generic (can handle any type of exchange with brazil), so we should make sure that is returns the proper direction. Therefore, we must check if BR is country_code1 or 2, and change directions accordingly.
Thanks for looking into this @alexanmtz ! Let me know if I wasn't clear.

Reply:
I see @olivierSemet , sorry, it is my confusion, this directions parte drives me crazy. I think I got you: depending on what country code Brazil it is when call the exchange function, it should handler the direction. I will address this.

But you can clarify me what was decided about the Brazil directions, because from source we had negatives.

So now we have:
```
"internacional": {
     "argentina":-0.00631053373,
     "paraguai":0.0,
     "uruguai":-60.5
}
```

This means that from BR-S -> UY should be negative?

Reply:
This should help you out:
https://github.com/tmrowco/electricitymap/blob/master/parsers/example.py#L127-L136
I suggest checking http://ons.org.br/pt/paginas/energia-agora/balanco-de-energia to be absolutely sure of which direction corresponds to positive/negative values in the data.